http/http_connection.hpp

```diff
@@ -330,9 +330,15 @@ class Connection :
         }
 
         std::string url(req->target());
-        if (boost::algorithm::contains(url,
-                                       "/system/LogServices/Dump/Entries/") &&
-            boost::algorithm::ends_with(url, "/attachment"))
+        // Check if the offload URL has system/resource dump entry ID
+        // System dumps ID starts with 'A<>'
+        // Resource dumps ID starts with 'B<>'
+        if ((boost::algorithm::contains(url,
+                                        "/system/LogServices/Dump/Entries/A") &&
```

Comment:
How does a Redfish user know A is system and B is resource?
Does an ID that gives some more information help? Can we fill out the Name/Description fields?